### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kamil-kielczewski**
Info about calling method with multi-arguments and info how to create regexp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,6 +364,24 @@ New characters:
 
 `j`, `<`, `>`, `=`, `"`, `/`
 
+### Calling method with more than one argument
+
+Calling method with more than one argument is non trivial - to do it you can use following [technique](https://stackoverflow.com/q/63601330/860099) (discovered by trincot) - for example  example: `"truefalse".replace("true","1")` can be written as follows:
```
Comment (aemkei) [inline README.md:369]: "method" should be "a method" or "methods"

Comment (aemkei) [reply README.md:369]: "you can use following" should be "you can use the following"

Comment (aemkei) [reply README.md:369]: "for example  example" should be "for example"

File: README.md [hunk 2]
```diff
@@ -364,6 +364,24 @@ New characters:
...
+
+```js
+["true"]["concat"]("1")["reduce"](""["replace"]["bind"]("truefalse"))
+```
```
Comment (aemkei) [inline README.md:373]: 👍 

This is a nice addition here!

File: README.md [hunk 3]
```diff
@@ -364,6 +364,24 @@ New characters:
...
+
+### Calling method with more than one argument in "flow way"
+
+To be able to call mehod (with multiple arguments) in right side on results of previous method you can use this [technique](https://stackoverflow.com/q/63604058/860099) (discovered by trincot) - for example: `"truefalse".replace("true","1").replace("false","0")` can be written as follows:
```
Comment (aemkei) [inline README.md:377]: "call mehod" should be "call a method"

File: README.md [hunk 4]
```diff
@@ -364,6 +364,24 @@ New characters:
...
+
+```js
+"truefalse"
+  ["split"]()["concat"]([["true"]["concat"]("1")])["reduce"](""["replace"]["apply"]["bind"](""["replace"]))
+  ["split"]()["concat"]([["false"]["concat"]("0")])["reduce"](""["replace"]["apply"]["bind"](""["replace"]))
+
+```
```
Comment (aemkei) [inline README.md:384]: This is super interesting, too! Thanks!

File: README.md [hunk 5]
```diff
@@ -391,8 +409,21 @@ When evaluating `function anonymous() { return this }`, we get the invocation co
...
----
+### Create regular expression object
+
+You can create regular expression e.g. `/pattern/g` as follows
+
+```js
+[]["fill"]["constructor"]("return RegExp")()("pattern","g")
+```
 
+wchich after remove comma (by using multi-arguments technique without `bind`ing) looks as follows
```
Comment (aemkei) [inline README.md:420]: "wchich" should be "which"
"after remove comma" should be "after removing the comma"

**Review by aemkei — CHANGES_REQUESTED** [review]
Thanks you! This is very helpful.

Once the typos are fixed, I will merge that into the main branch.

File: README.md (hunk 2 above)
Comment (kamil-kielczewski) [reply README.md:373]: fixed

File: README.md (hunk 1 above)
Comment (kamil-kielczewski) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (kamil-kielczewski) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (kamil-kielczewski) [reply README.md]: fixed

**Review by aemkei — APPROVED** [review]

**Pull request merged**